### DISCUSSION TITLE
M109.0.5414.172 - Final M109 Release

### DISCUSSION THREAD

**Discussion by Alex313031** [Alex313031/thorium-legacy#21] [Announcements]
[UPDATE]: It turns out they made 3 more minor versions, with the true final one being 109.0.5414.172. I also screwed up Widevine on the last .169 release. So I am rebuilding and uploading them here. I also never completed the release notes for the previous one, so see the bottom for changes between 159 - 172

 - Chromium bump from __109.0.5414.159__ > __109.0.5414.169__ and now to > __109.0.5414.172__
 - This is the LAST version of M109. It includes fixed for two major security vulnerabilities in libwebp and libvpx. See the new [security policy](https://github.com/Alex313031/thorium/security/policy) for info about submitting security bugs, and a list of fixed vulnerabilities (which will be updated henceforth). If you use any of my Electron apps, those were also recently fixed.
 - Next version will be based on [Supermium](https://github.com/win32ss/supermium) M118
 - Backported these patches from latest Thorium (Includes history of 159 - 169 & 169-172): 
 > - F

<hr /><em>This discussion was created from the release <a href='https://github.com/Alex313031/thorium-win7/releases/tag/M109.0.5414.172'>M109.0.5414.172 - Final M109 Release</a>.</em>

**Comment by libreom** [comment]
Hi when will thorium-win7 based on supermium be released?

**Reply by Conflituoso** [reply]
Hi.

There won't be a WIN32 version of the 172?

**Reply by Conflituoso** [reply]
Thanks a lot for the WIN32 version!!!

**Comment by Ru-Sky-Gin** [comment]
Thanks bro for the support on Win7!
Finally found you!

**Comment by HYDENKO** [comment]
🤩Thank you very much for making **SSE2** builds ✨
It saved my friend's 2005 netbook (and not only) ♥️

**Comment by Dratas** [comment]
Hello just mine 5cents,but when trying log-in to discord.com after i write user/pass combos,lastest build thorium SSE3 for win 7 M109.0.5414.172,shows for me STATUS_ACCESS_VIOLATION error,what to do?not tried normal release and SSE2.

**Reply by Kame03** [reply]
I have the same issue with Discord.

**Reply by Kame03** [reply]
I've opened a bug report: https://github.com/Alex313031/thorium/issues/442

**Comment by Alex313031** [comment]
@Dratas @retapich @Ru-Sky-Gin @Kame03 This was fixed in the latest M122 release.

**Reply by Stepman123** [reply]
Version 122 is too heavy for older equipment, so it cannot replace 109.

**Comment by Kame03** [comment]
Great thanks for letting me know.    ---- On Wed, 03 Apr 2024 08:35:49 +0200  Alex ***@***.***> wrote ----  
@Dratas @retapich @Ru-Sky-Gin @Kame03 This was fixed in the latest M122 release.
